packages/flutter_tools/lib/src/commands/create_base.dart

```diff
@@ -530,7 +529,7 @@ abstract class CreateBase extends FlutterCommand {
         engineVersion: globals.flutterVersion.engineRevision,
         fileSystem: globals.fs,
         flutterRootDir: globals.fs.directory(Cache.flutterRoot),
-        outputDir: globals.fs.directory(getBuildDirectory()),
```

Comment:
Yeah thanks for the catch,  I didn't mean to convert this to be ready for review yet, i was testing the presubmit checks if swapping that out would cause any failures.  I still need to determine why swapping this out worked but I got caught up with the test question